plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/messages.properties

```diff
@@ -19,6 +19,8 @@ CANNOT_DETERMINE_EXECUTION_CONTEXT=Cannot determine server execution context
 SERVER_ALREADY_RUNNING=Server is already running
 SERVER_ALREADY_RUNNING_IN_MODE=Server is already running in "{0}" mode
 UNABLE_TO_LAUNCH=Unable to launch App Engine server
+STALE_RESOURCES_DETECTED=Stale resources detected
+STALE_RESOURCES_LAUNCH_CONFIRMATION=Some recently-changed resources have not been published yet.  Continue with launch?
```

Comment:
_Recently changed_.  These stale resources are usually from the user having saved their unsaved changes, which haven't been published yet.